packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Fixed `onBlur` event to not be triggered when selecting an option
```

Comment:
Does this need to be a bit more specific that this was done _just_ for mobile?

Reply:
Updated here https://github.com/cerner/terra-core/pull/2904/commits/09eb6be42fd6ff175b3d324f8d5ccbb9ca1583ec